packages/sail_ui/lib/style/style_values.dart

```diff
@@ -23,6 +23,7 @@ abstract class SailStyleValues {
   static const lightWeight = FontWeight.w300;
   static const regularWeight = FontWeight.w400;
   static const mediumWeight = FontWeight.w500;
+  static const boldWeight = FontWeight.w700;
```

Comment:
```suggestion
  static const boldWeight = FontWeight.w500;
```

Reply:
The reason I haven't used w700 is that is reeeally intense

Reply:
Reverted the change, I added it because the medium weight did not appear any thicker than the regular (400) weight and I was trying to get a bold look. Maybe semibold could work?